### ISSUE TITLE
Shell Center Row - Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality - (2036570599)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7833]
### Violation: 
 Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality 

### WCAG Reference: 
- [1.4.10 Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) 

### Severity: 
 6 

### Media Type: 
 Typography 

--- 
### Areas for Remediation: 
- Title: Shell Center Row - Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality - (2036570599) 
- Module: Shell Center Row 
**Pattern:** Reflow - Content is partially available.]

**Issue**
The content does not resize properly to accommodate smaller screen sizes, and content is cut-off or partially available when the page has a viewport width of 320 CSS pixels.

**Occurrences**
29 Shell and Shell Panel
30 Shell Center Row

**User Impact**
Users with low vision who need to resize text will be prevented from accessing this content.
 
**Suggestion**
Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality. When the page has a viewport width of 320 CSS pixels, all content and functionality must still be available without requiring horizontal scrolling.

Moving content to an accessible show/hide control, such as a hamburger menu, is acceptable.

Content that requires two dimensions to be understood, such as a map or data table, is exempt from this requirement.
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
I think we can just deprecate this component.

**Comment by geospatialem** [comment]
`calcite-shell-center-row` will be deprecated in the near future once https://github.com/Esri/calcite-design-system/issues/7513 is fulfilled. Closing this issue out as it will be invalid with the deprecation and eventual removal of the component.
